### PR TITLE
W-13808883-rtf-kubernetes-crd-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by sarathecoubian — DISMISSED** [review]

**Review by martincousido — DISMISSED** [review]

**Review by Crispy-Salesforce — DISMISSED** [review]

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 1]
```diff
@@ -0,0 +1,179 @@
+== Customize Mule Application Kubernetes Resources
+
+Anypoint Runtime Fabric enables you to customize your Mule application Kubernetes resources through the custom resource `kubernetestemplates.rtf.mulesoft.com` template.
+
+Name the `kubernetestemplate` resource as `mule-application`. You can define the template in two different namespaces:
+
+* Applications namespace +
+The template is applied to all Mule applications deployed to the given namespace.
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
The template is applied to all Mule apps deployed to the given namespace.
```
Use "app" except when "Application" is part of a license, permission, or product name

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 2]
```diff
@@ -0,0 +1,179 @@
+== Customize Mule Application Kubernetes Resources
+
+Anypoint Runtime Fabric enables you to customize your Mule application Kubernetes resources through the custom resource `kubernetestemplates.rtf.mulesoft.com` template.
+
+Name the `kubernetestemplate` resource as `mule-application`. You can define the template in two different namespaces:
+
+* Applications namespace +
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
* Apps namespace +
```

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 3]
```diff
@@ -0,0 +1,179 @@
+== Customize Mule Application Kubernetes Resources
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
== Customize Mule App Kubernetes Resources
```

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+Anypoint Runtime Fabric enables you to customize your Mule application Kubernetes resources through the custom resource `kubernetestemplates.rtf.mulesoft.com` template.
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
Anypoint Runtime Fabric enables you to customize your Mule app Kubernetes resources through the custom resource `kubernetestemplates.rtf.mulesoft.com` template.
```

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+Name the `kubernetestemplate` resource as `mule-application`. You can define the template in two different namespaces:
+
+* Applications namespace +
+The template is applied to all Mule applications deployed to the given namespace.
+
+* Runtime Fabric agent namespace +
+The template is applied to all Mule applications managed by the given Runtime Fabric agent.
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
The template is applied to all Mule apps managed by the given Runtime Fabric agent.
```

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+
+The `kubernetestemplate` resource applies only to new Mule application deployments after the resource creation. It does not affect existing deployments. If you redeploy a Mule application after creating or updating the `kubernetestemplate` the template is applied.
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
The `kubernetestemplate` resource applies only to new Mule app deployments after the resource creation. It does not affect existing deployments. If you redeploy a Mule app after creating or updating the `kubernetestemplate`, the template is applied.
```

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+The `kubernetestemplate` resource enables you to configure the following elements of your applications pods, deployments and services objects:
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
The `kubernetestemplate` resource enables you to configure the following elements of your apps pods, deployments, and services objects:
```

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+[source,copy]
+----
+kubectl get crd kubernetestemplates.rtf.mulesoft.com -o yaml
+----
+
+
+== Examples
+
+Configuring `labels` at `pod`, `deploy` and `service` objects:
```
Comment (dmerlob) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc]:
```suggestion
Configuring `labels` at `pod`, `deploy`, and `service` objects:
```

**Review by dmerlob — DISMISSED** [review]

**Review by dmerlob — APPROVED** [review]

**Pull request merged**